lib/appsignal.rb

```diff
@@ -301,6 +294,15 @@ def start_stdout_logger
         "appsignal: #{msg}\n"
```

Comment:
Not sure how this is usually handled. Maybe it's normal that timestamps are already appended on most platforms?